caffe2/operators/conv_op_cudnn.cc

```diff
@@ -54,7 +66,8 @@ class CudnnConvOpBase : public ConvPoolOpBase<CUDAContext> {
         deterministic_(
             OperatorBase::GetSingleArgument<int>("deterministic", 0)),
         cudnn_state_(OperatorBase::GetSingleArgument<int>("cudnn_state", 0)),
-        force_algo_(OperatorBase::GetRepeatedArgument<int>("force_algo", vector<int>{-1,-1,-1})) {
+        force_algo_(OperatorBase::GetRepeatedArgument<int>("force_algo", vector<int>{-1,-1,-1})),
+        enable_tensor_core_(OperatorBase::GetSingleArgument<bool>("enable_tensor_core", 0)) {
```

Comment:
So, do we want to enable tensor core by default? Is it always going to be faster?

If so, I think we should enable it by default, and then do

enable_tensor_core_ &&= TensorCoreAvailable(device_id);

or something.